pkg/build/api/helpers.go

```diff
@@ -17,3 +17,32 @@ func BuildToPodLogOptions(opts *BuildLogOptions) *kapi.PodLogOptions {
 		LimitBytes:   opts.LimitBytes,
 	}
 }
+
+// FilterBuilds returns array of builds that satisfies predicate function.
+func FilterBuilds(builds []Build, predicate func(Build) bool) []Build {
```

Comment:
I'm not sure about this, because in Java we have a `filter()` method that accepts the predicate function: https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#filter-java.util.function.Predicate-

May be it was a bad idea to pull something from my Java background but I thought that invented something by myself is worst than using something that already used by other people. At least, the approach that is used in Java may be more understandable for some people who knows this convention.

I don't insist on it, so if you still think that I need to change the name, I may rename.